src/components/App.tsx

```diff
@@ -190,6 +190,11 @@ import { actionToggleViewMode } from "../actions/actionToggleViewMode";
 
 export const IsMobileContext = React.createContext(false);
 export const useIsMobile = () => useContext(IsMobileContext);
+export const ExcalidrawContainerContext = React.createContext<HTMLDivElement | null>(
```

Comment:
```suggestion
const IsMobileContext = React.createContext(false);
export const useIsMobile = () => useContext(IsMobileContext);
const ExcalidrawContainerContext = React.createContext<HTMLDivElement | null>(
```

since we're exporting the hooks for some reason, let's make the contexts private